apps/ledger-live-mobile/src/components/RootNavigator/OnboardingNavigator.tsx

```diff
@@ -81,12 +80,6 @@ function OnboardingCarefulWarning(
       backgroundColor="background.main"
     >
       <OnboardingCarefulWarningStack.Navigator>
-        <OnboardingCarefulWarningStack.Screen
-          name={ScreenName.OnboardingModalWarning}
-          component={OnboardingWarning}
-          options={{ title: "", ...options }}
-          // initialParams={props.route.params}
-        />
         <OnboardingCarefulWarningStack.Screen
           name={ScreenName.OnboardingModalSyncDesktopInformation}
           component={OnboardingSyncDesktopInformation}
```

Comment:
Do we have the same issue for others "Drawers" ?

Reply:
Well, not that I know of, I removed `OnboardingModalSyncDesktopInformation` drawer as it was unused and just got rid of the `OnboardingCarefulWarning` component as well as it was empty.